### PR TITLE
Add the VS Code extension license

### PR DESCRIPTION
**Pull request by praneesha**
Add the VS Code extension license

### CODE REVIEW

**Review by axewilledge — APPROVED** [review]

**Comment by keizer619** [comment]
I think we should keep the License in the public [vscode repo](https://github.com/wso2/ballerina-plugin-vscode) and mention that in the website

**Review by gigara — APPROVED** [review]

File: en/docs/license.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Ballerina VS Code extesnion license 
+
+WSO2 grants you a worldwide, non-exclusive right to copy and use the Ballerina Visual Studio Code extenstion in
```
Comment (anupama-pathirage) [inline en/docs/license.md:3]: Typo `extenstion`

**Comment by praneesha** [comment]
> vscode repo

The VS Code repo has the Apache license[1]. Therefore, linked to the license added in the VS Code marketplace[2].

[1] https://github.com/wso2/ballerina-plugin-vscode/blob/main/LICENSE
[2] https://marketplace.visualstudio.com/items/WSO2.ballerina/license

File: en/docs/license.md (hunk 1 above)
Comment (praneesha) [reply en/docs/license.md:3]: Fixed in https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/21/files.

**Pull request merged**